### PR TITLE
Add timestamp fields to models

### PR DESCRIPTION
**Pull request by reinbach**
Do we want to add the created/modified field to the admin section fir each of the models affected?

The following models are still outstanding timestamps;

partners:
- WorkspaceFileType
- BankDetails
- FileType
- GovernmentIntervention
- GovernmentInterventionResult
- GovernmentInterventionResultActivity
- DirectCashTransfer
- PCA
- RAMIndicator
- PCAFile
- GwPCALocation
- PCASectorGoal
- IndicatorDueDates
- AuthorizedOfficer

reports:
- Quarter
- CountryProgramme
- ResultType
- Goal
- Unit
- GatewayType

location:
- CartoDBTable

Let me know if you want timestamps added to these models as well.

### CODE REVIEW

**Comment by robertavram** [comment]
putting on hold: depending on https://github.com/unicef/etools/pull/943

File: EquiTrack/reports/models.py [hunk 1]
```diff
@@ -214,6 +216,8 @@ class Result(MPTTModel):
 
     hidden = models.BooleanField(default=False)
     ram = models.BooleanField(default=False)
+    created = AutoCreatedField(_('created'))
```
Comment (ntrncic) [inline EquiTrack/reports/models.py]: do we need this if it's timestamped?

Comment (reinbach) [reply EquiTrack/reports/models.py]: The `Result` class is based on `MPTTModel`, so not able to make use of the `TimeStampedModel` easily. It was simpler to just add the fields

**Comment by ntrncic** [comment]
please resolve conflicts and update migrations

**Pull request merged**